src/components/scene/inspector.js

```diff
@@ -59,7 +59,7 @@ module.exports.Component = registerComponent('inspector', {
    * <ctrl> + <alt> + i keyboard shortcut.
    */
   onKeydown: function (evt) {
-    var shortcutPressed = evt.keyCode === 73 && evt.ctrlKey && evt.altKey;
+    var shortcutPressed = evt.keyCode === 73 && (evt.ctrlKey && evt.altKey || evt.getModifierState('AltGraph'));
```

Comment:
AltGraph is deprecated as per https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/getModifierState

Reply:
Hmm I see. There doesn't seem to be another way to detect ctrl + alt on an US international keyboard. Some sources report that `crtlKey` and `altKey` should both be true but that doesn't seem to be the case on both Firefox and Chrome.

The only other solution I can think of right now is to use `evt.key === 'í'`, as í is the key returned when pressing ctrl + alt + i on a US international keyboard. This, however, seems quite layout dependent.

Reply:
The link has been updated and AltGraph no longer seems deprecated as far as I can see.